includes/class-wc-gateway-ppec-client.php

```diff
@@ -690,8 +696,14 @@ protected function _get_details_from_order( $order_id ) {
 		}
 		$shipping_address->setCountry( $shipping_country );
 
+		if ( ! $old_wc ) {
+			$shipping_address->setPhoneNumber( $old_wc ? '' : $order->get_billing_phone() );
```

Comment:
Why do you only do this if the WC version is new? I can't see anything related to the `phone` field in the WooCommerce changelog.

Reply:
Good call – none of the fields needed to be omitted for old WC versions. I was thrown off by the changes for CRUD order and customer classes new in 3.0, and didn't realize we should just be accessing the properties directly on the objects. Fixed in https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/pull/477/commits/8a67490a719d5211e0fc701c367c9b8b5ed17b57